### PR TITLE
Fix multicluster labels

### PR DESCRIPTION
**Pull request by phillebaba**
More fixes like #752 for multi cluster labels which can cause issues if labels overlap.

### CODE REVIEW

**Comment by arajkumar** [comment]
@phillebaba Tests are failing, please fix it.

File: alerts/system_alerts.libsonnet [hunk 1]
```diff
@@ -28,9 +28,9 @@
             // this is normal and an expected error, therefore it should be
             // ignored in this alert.
             expr: |||
-              (sum(rate(rest_client_requests_total{code=~"5.."}[5m])) by (instance, job, namespace)
+              (sum(rate(rest_client_requests_total{code=~"5.."}[5m])) by (%(clusterLabel)s, instance, job, namespace)
                 /
-              sum(rate(rest_client_requests_total[5m])) by (instance, job, namespace))
+              sum(rate(rest_client_requests_total[5m])) by (%(clusterLabel)s, instance, job, namespace))
               > 0.01
             |||,
```
Comment (arajkumar) [inline alerts/system_alerts.libsonnet:35]:
```suggestion
            ||| % $._config,
```

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: alerts/kubelet.libsonnet [hunk 2]
```diff
@@ -52,11 +52,11 @@
             // Some node has a capacity of 1 like AWS's Fargate and only exists while a pod is running on it.
             // We have to ignore this special node in the KubeletTooManyPods alert.
             expr: |||
-              count by(node) (
+              count by(%(clusterLabel)s, node) (
                 (kube_pod_status_phase{%(kubeStateMetricsSelector)s,phase="Running"} == 1) * on(instance,pod,namespace,cluster) group_left(node) topk by(instance,pod,namespace,cluster) (1, kube_pod_info{%(kubeStateMetricsSelector)s})
```
Comment (arajkumar) [inline alerts/kubelet.libsonnet:56]:
```suggestion
                (kube_pod_status_phase{%(kubeStateMetricsSelector)s,phase="Running"} == 1) * on(instance,pod,namespace,%(clusterLabel)s) group_left(node) topk by(instance,pod,namespace,%(clusterLabel)s) (1, kube_pod_info{%(kubeStateMetricsSelector)s})
```

**Review by arajkumar — CHANGES_REQUESTED** [review]

**Comment by phillebaba** [comment]
@arajkumar thanks for the help, should be all set now!

**Review by metalmatze — APPROVED** [review]

**Comment by metalmatze** [comment]
Thanks!

**Pull request merged**